package.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<package>
+<package format="2">
```

Comment:
please do package conversion to format 2 in a separate commit so that it can be submitted upstream easily. https://github.com/ros-perception/pcl_conversions/issues/26#issue-224155516